chia/farmer/farmer_api.py

```diff
@@ -443,139 +582,10 @@ async def respond_signatures(self, response: harvester_protocol.RespondSignature
-            full_node_request.quality_string
-        ]
-        request = harvester_protocol.RequestSignatures(
-            plot_identifier,
-            challenge_hash,
-            sp_hash,
-            [full_node_request.foliage_block_data_hash, full_node_request.foliage_transaction_block_hash],
-        )
-
-        msg = make_msg(ProtocolMessageTypes.request_signatures, request)
-        await self.farmer.server.send_to_specific([msg], node_id)
-
-    @api_request()
-    async def farming_info(self, request: farmer_protocol.FarmingInfo) -> None:
-        self.farmer.state_changed(
-            "new_farming_info",
-            {
-                "farming_info": {
-                    "challenge_hash": request.challenge_hash,
-                    "signage_point": request.sp_hash,
-                    "passed_filter": request.passed,
-                    "proofs": request.proofs,
-                    "total_plots": request.total_plots,
-                    "timestamp": request.timestamp,
-                }
-            },
-        )
-
-    @api_request(peer_required=True)
-    async def respond_plots(self, _: harvester_protocol.RespondPlots, peer: WSChiaConnection) -> None:
-        self.farmer.log.warning(f"Respond plots came too late from: {peer.get_peer_logging()}")
-
-    @api_request(peer_required=True)
-    async def plot_sync_start(self, message: PlotSyncStart, peer: WSChiaConnection) -> None:
-        await self.farmer.plot_sync_receivers[peer.peer_node_id].sync_started(message)
-
-    @api_request(peer_required=True)
-    async def plot_sync_loaded(self, message: PlotSyncPlotList, peer: WSChiaConnection) -> None:
-        await self.farmer.plot_sync_receivers[peer.peer_node_id].process_loaded(message)
-
-    @api_request(peer_required=True)
-    async def plot_sync_removed(self, message: PlotSyncPathList, peer: WSChiaConnection) -> None:
-        await self.farmer.plot_sync_receivers[peer.peer_node_id].process_removed(message)
-
-    @api_request(peer_required=True)
-    async def plot_sync_invalid(self, message: PlotSyncPathList, peer: WSChiaConnection) -> None:
-        await self.farmer.plot_sync_receivers[peer.peer_node_id].process_invalid(message)
-
-    @api_request(peer_required=True)
-    async def plot_sync_keys_missing(self, message: PlotSyncPathList, peer: WSChiaConnection) -> None:
-        await self.farmer.plot_sync_receivers[peer.peer_node_id].process_keys_missing(message)
-
-    @api_request(peer_required=True)
-    async def plot_sync_duplicates(self, message: PlotSyncPathList, peer: WSChiaConnection) -> None:
-        await self.farmer.plot_sync_receivers[peer.peer_node_id].process_duplicates(message)
-
-    @api_request(peer_required=True)
-    async def plot_sync_done(self, message: PlotSyncDone, peer: WSChiaConnection) -> None:
-        await self.farmer.plot_sync_receivers[peer.peer_node_id].sync_done(message)
+        return None
```

Comment:
not sure if this None is needed.

Reply:
It's not needed, but recommended for functions which actually return stuff (pep8), if desired i can remove it ofc

Reply:
should be fine then, was not sure if mypy cared or not.